releases/v0.2.0-preview.2.md

```
@@ -0,0 +1,95 @@
+<!--
+Generated by ./eng/release prepare.
+Version: 0.2.0-preview.2
+Date: 2026-07-02
+-->
+# Release 0.2.0-preview.2
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)